doc/netplan-yaml.md

```diff
@@ -105,6 +105,12 @@ Match devices by MAC when setting options like: `wakeonlan` or `*-offload`.
     > A sequence of globs is supported, any of which must match.
     > Matching on driver is *only* supported with networkd.
 
+  - **pciid** (scalar) – since **0.107**
+
+    > The PCI ID, corresponding to the `ID_PATH` udev properity. (`0000:98:00.1`)
+    > Matching on pci id is *only* supported with networkd.
+    > `udevadm info /sys/class/net/DEVICE_NAME`
```

Comment:
nitpick (non-blocking): Version needs to be bumped.

question: Also, is there any possibility of doing the same for the NetworkManager backend? The `match` stanza is a very central piece of Netplan's configuration and we should try not to diverge between the backends here.